src/views/statutory/VotesAmounts.vue

```diff
@@ -82,16 +82,16 @@
 import { mapGetters } from 'vuex'
 
 /**
- * Takes a list of antennas, and returns an object with two properties, CSV content and a filename
+ * Takes a list of antennas, and returns an object with two properties, TXT content and a filename
```

Comment:
better to rename it to TSV then: https://en.wikipedia.org/wiki/Tab-separated_values
also, last time I remember it required semicolon-separated values or something like that, can you double check?

Reply:
This is what Heiko sent me;

> For the votes: I checked the export and tried uploading it in the system, but it was unsuccessful. My guess is that it’s because the file is comma-separated instead of tab-separated.
> If it helps: here’s the instructions from statutoryvote: “File must be tab-separated (in Excel: "File-> Save as -> Text (Tab delimited)" )/ There must be no header lines (i.e. data start from first line)/
> There should be 2 columns: 1st column: 3-letter code, 2nd column: number of votes (only numbers, e.g. 0 instead of "-"!)/ Delete other columns, do not leave any empty columns or lines.”

If I follow the instructions in Excel, it saves it as a TXT so I just used that

Reply:
The delegates export uses semicolons. Not sure why the two are different, but they are